ckanext/publications_qld_theme/templates/package/resource_read.html

```diff
@@ -100,7 +101,7 @@
       <h2>{{ _('Data Dictionary') }}</h2>
       <table class="table table-striped table-bordered table-condensed" data-module="table-toggle-more">
         <thead>
-          {% block resouce_data_dictionary_headers %}
+          {% block resource_data_dictionary_headers %}
```

Comment:
Is this function https://github.com/qld-gov-au/ckanext-publications-qld-theme/blob/bf3623c1b58a11419889e1aad73dbd0fcf8468e2/ckanext/publications_qld_theme/plugin.py#L97 a bit expensive, when we should be asking the database for 'last' record not all and programmatically loop through them all...?

Reply:
We already do filter it at the database level, requesting only revisions with expiry in the year 9999. There should only be zero or one result; the loop is just defensive programming.